src/stock-price-changes/stock-price-changes.html

```diff
@@ -41,13 +41,15 @@ <h2 ng-hide="vm.facility && vm.program">{{'stockPriceChanges.title' | message}}<
         <tr>
             <th>{{'stockCardSummaryList.productCode' | message}}</th>
             <th>{{'stockCardSummaryList.product' | message}}</th>
+            <th>{{'stockCardSummaryList.UnitPrice' | message}}</th>
```

Comment:
Naming convention for translation strings is to have a key start lowercase after each section (dot). Can you rename it? This would also need to be changed in the messages_en.json file.

```suggestion
            <th>{{'stockCardSummaryList.unitPrice' | message}}</th>
```